src/ChunkMap.cpp

```diff
@@ -740,17 +740,17 @@ bool cChunkMap::DoWithChunkAt(Vector3i a_BlockPos, std::function<bool(cChunk &)>
 
 
 
-void cChunkMap::WakeUpSimulators(int a_BlockX, int a_BlockY, int a_BlockZ)
+void cChunkMap::WakeUpSimulators(const Vector3i & a_Block)
 {
 	cCSLock Lock(m_CSChunks);
 	int ChunkX, ChunkZ;
-	cChunkDef::BlockToChunk(a_BlockX, a_BlockZ, ChunkX, ChunkZ);
+	cChunkDef::BlockToChunk(a_Block.x, a_Block.z, ChunkX, ChunkZ);
```

Comment:
Right, sorry, I missed the fact that there's no Y coord.
Still, perhaps consider this signature, as an overload (not replacement) for ease of use?
```cpp
cChunkDef::BlockToChunk(
  const Vector3i & a_BlockCoords,  // The Y coord is ignored
  int & a_OutChunkX,
  int & a_OutChunkZ
);
```